src/components/Modal/index.tsx

```diff
@@ -96,6 +97,7 @@ export default function Modal({
   className,
   children,
   transition = true,
+  zIndex = 100,
```

Comment:
Why is 100?

Reply:
zindex of the old code is 100.  => default value is 100.     I add props so that we can set dynamic for popup. if not set => zindex will not effect
<img width="436" alt="image" src="https://user-images.githubusercontent.com/33005392/174517822-51ba27a5-f155-4618-b816-099f4fd882a4.png">